src/utils/statusBarUtil.ts

```diff
@@ -10,7 +10,8 @@ export class StatusBarUtil {
   private static promises: { [text: string]: Thenable<any> } = {}
 
   private static statusBarItem = vscode.window.createStatusBarItem(
-    vscode.StatusBarAlignment.Left
+    vscode.StatusBarAlignment.Left,
+    100
```

Comment:
100 ? Can you make it lower to make it so it won't have more importance than the Git stuff